examples/application/main.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"../../iis"
+)
+
+func main() {
+	log.Printf("Starting test...")
```

Comment:
IME in Golang it's generally easiest to have a `run` function which can return an error, which gets called from the main function - since this allows you to just return wrapped errors instead - as such could we make this:

```
func main() {
	log.Printf("Starting example...")

	if err := run(); err != nil {
		panic(err)
	}
}

func run() error {
 ...
```